server/container/guice/guice-common/src/main/java/org/apache/james/jmap/JMAPCommonModule.java

```diff
@@ -71,6 +73,7 @@ protected void configure() {
 
         bindConstant().annotatedWith(Names.named(AccessTokenRepository.TOKEN_EXPIRATION_IN_MS)).to(DEFAULT_TOKEN_EXPIRATION_IN_MS);
         bind(AccessTokenManager.class).to(AccessTokenManagerImpl.class);
+        bind(AttachmentManager.class).to(StoreAttachmentManager.class).in(Scopes.SINGLETON);
```

Comment:
Usually we are giving the scope to the implementation, then binding it to an interface.